src/rSharp.h

```diff
@@ -144,6 +144,7 @@ extern "C" {
 RSharpGenericValue ConvertToRSharpGenericValue(SEXP s);
 RSharpGenericValue** sexp_to_parameters(SEXP args);
 SEXP ConvertToSEXP(RSharpGenericValue& value);
+char_t* MergeLibPath(char** libPath, char* additionalPath);

```

Comment:
There is lots of cleanup required in this file. In general, we should not expose the prototypes for functions that are not 'public'.

Put this prototype in the rSharp.c file toward the top.